nupic/frameworks/opf/metrics.py

```diff
@@ -190,6 +190,8 @@ def getModule(metricSpec):
     return MetricMAPE(metricSpec)
   elif metricName == 'multi':
     return MetricMulti(metricSpec)
+  elif metricName == 'negLL':
```

Comment:
There's not particular reason to use a short name. Can we use a more descriptive name for the metric? Perhaps "negativeLogLikelihood"?